src/Components/App/App.js

```diff
@@ -18,10 +18,19 @@ const App = () => {
   const [activities, setActivities] = useState(activityData);
   const [filteredActivities, setFilteredActivities] = useState([]);
 
+  useEffect(() => {
```

Comment:
LOVE HOW YOU SET THIS UP TO BE A SEAMLESS TRANSITION TO FETCHING OUR DATA

Reply:
I'm 'hook' d